docs/data/deployments.yml

```diff
@@ -221,3 +221,12 @@
   dask: dask-jobqueue
   jupyter: Single user notebook servers
   kind: hpc
+  
+- name: JASMIN Cluster-as-a-Service
+  description: Dynamic, user-provisioned Pangeo clusters running on the JASMIN Cloud. Supported by UK Science and Technology Facilities Council.
+  link: http://www.jasmin.ac.uk/
+  platform: OpenStack Cloud
+  dask: dask-kubernetes
+  jupyter: JupyterHub
+  access: Requires access to a JASMIN Cloud tenancy, available primarily for NCAS-affiliated projects.
```

Comment:
@kaedonkers 

This should say NERC rather than NCAS. My bad.

Reply:
👍